pinot-core/src/main/java/org/apache/pinot/core/segment/processing/mapper/SegmentMapper.java

```diff
@@ -62,22 +64,20 @@
  */
 public class SegmentMapper {
   private static final Logger LOGGER = LoggerFactory.getLogger(SegmentMapper.class);
-
-  private List<RecordReaderFileConfig> _recordReaderFileConfigs;
-  private List<RecordTransformer> _customRecordTransformers;
   private final SegmentProcessorConfig _processorConfig;
   private final File _mapperOutputDir;
-
   private final List<FieldSpec> _fieldSpecs;
   private final boolean _includeNullFields;
   private final int _numSortFields;
-
   private final CompositeTransformer _recordTransformer;
   private final TimeHandler _timeHandler;
   private final Partitioner[] _partitioners;
   private final String[] _partitionsBuffer;
   // NOTE: Use TreeMap so that the order is deterministic
   private final Map<String, GenericRowFileManager> _partitionToFileManagerMap = new TreeMap<>();
+  AdaptiveSizeBasedWriter _adaptiveSizeBasedWriter;
```

Comment:
`private final` if we expect this to be immutable once initialized? Otherwise, let's put `private`

Reply:
missed putting the access modifier. Thank you for pointing it out. added.